### DISCUSSION TITLE
Windows Domoticz support

### DISCUSSION THREAD

**Discussion by JoostdeJonge123** [addiejanssen/domoticz-solaredge-modbustcp-plugin#32] [Q&A] [answered]
I'm running Domoticz already for many years on a Windows machine. Does somebody know if it possible to use the Modbus plugin on Windows? At the moment I'm running into :  "Error: Solaredge:  ----> Line 48 in '()', function Unknown"

Any ideas? 

Thanks!

**Comment by addiejanssen** [comment]
Hi Joost,

The plugin depends on the [solaredge_modbus library](https://github.com/nmakel/solaredge_modbus) (version 0.6.2 for now).
The error message indicates that the library is not installed or can not be found by Domoticz.
Did you install that library?

Kind regards,

Addie

**Comment by JoostdeJonge123** [comment, marked as answer]
Hi Addie,
I think something went wrong with the modbus library version, it seems to be working now.
Thanks for your quick response.

Regards, Joost
